.devcontainer/vmr-source-build/README.md

```diff
@@ -0,0 +1,27 @@
+<!--
+########  ########    ###    ########     ######## ##     ## ####  ######
+##     ## ##         ## ##   ##     ##       ##    ##     ##  ##  ##    ##
+##     ## ##        ##   ##  ##     ##       ##    ##     ##  ##  ##
+########  ######   ##     ## ##     ##       ##    #########  ##   ######
+##   ##   ##       ######### ##     ##       ##    ##     ##  ##        ##
+##    ##  ##       ##     ## ##     ##       ##    ##     ##  ##  ##    ##
+##     ## ######## ##     ## ########        ##    ##     ## ####  ######
+-->
+
+This Codespace can help you debug the source build of .NET. In case you have run this from a
+`dotnet/installer` PR branch, it will contain the VMR (`dotnet/dotnet`) checked out into
+`/workspaces/dotnet` with the PR changes pulled into it. You can then attempt to source-build
+the VMR which is what the VMR leg in the installer PR build doing. This build takes about 45
+minutes and, after completion, produces an archived .NET SDK in `/workspaces/artifacts/x64/Release`.
+
+To build the VMR, run following:
+```bash
+cd /workspaces/dotnet
+unset RepositoryName
+./build.sh --online --clean-while-building
```

Comment:
Given this is intended to debug source build, I don't think this should use the clean-while-building option.
```suggestion
./build.sh --online
```